src/main/Extensions/ApplicationSearch/ApplicationSearch.ts

```diff
@@ -36,7 +36,17 @@ export class ApplicationSearch implements Extension {
 
     public isSupported(): boolean {
         const checks: Record<OperatingSystem, () => boolean> = {
-            Linux: () => ["cinnamon", "gnome"].includes(this.environmentVariableProvider.get("XDG_SESSION_DESKTOP")),
+            Linux: () => {
+                const supportedLinuxDesktopEnvironments: LinuxDesktopEnvironment[] = [
+                    "GNOME",
+                    "KDE",
+                    "Cinnamon",
+                    "MATE",
+                    "XFCE",
+                    "Pantheon",
+                ];
+                return supportedLinuxDesktopEnvironments.includes(this.linuxDesktopEnvironmentResolver.resolve());
```

Comment:
If this works on every desktop environment, then we don't need this list here, right?